apps/wing/src/index.ts

```diff
@@ -41,10 +41,13 @@ async function main() {
       "Output directory",
       join(process.cwd(), "target")
     )
-    .option(
-      "-t, --target <target>",
-      "Target platform (options: 'tf-aws', 'tf-azure', 'tf-gcp', 'sim')",
-      "tf-aws"
+    .addOption(
+      new Option(
+        "-t, --target <target>",
+        "Target platform"
+      )
+      .choices(["tf-aws", "tf-azure", "tf-gcp", "sim"])
+      .makeOptionMandatory()
```

Comment:
Have you tried using `.requiredOption()`? It worked for me before. I think the commander.js documentation you linked to (thank you for that!) is a bit misleading in regards to required options.

Reply:
Yea the issue with that option is you can’t add the “choices” afaik.